components/Slider/src/MDCSlider.h

```diff
@@ -221,6 +221,20 @@ IB_DESIGNABLE
  */
 @property(nonatomic, assign) BOOL shouldDisplayDiscreteValueLabel;
 
+/**
+ The color of the discrete value label's text.
+
+ Defaults and resets to white.
+ */
+@property(nonatomic, strong, null_resettable) UIColor *valueLabelTextColor;
+
+/**
+ The color of the discrete value label's background.
+
+ Defaults and resets to blue.
```

Comment:
It's a blue, I don't like documenting exactly which one since then we'd have to do breaking changes to update it later. It was not documented before - I can just call it "a default color" if you prefer.

Reply:
up to you, as a developer i'd like to know what color i'm getting and defaulting to rather than reading the source code.

Reply:
I think if it's going to be contentious, I'd prefer to just remove the default documentation altogether. That way nobody can argue when blue becomes green and we don't explicitly need to support a specific blue.  If we were to change the default blue in the future to something more accessible, it would require deprecation.